README.md

```diff
@@ -1,6 +1,8 @@
-# Academy Resident Information API
+# LBH Academy Resident Information API
 
-Base API is a boilerplate code for being reused for new APIs for LBH
+Academy Resident Information API retrieves data from the Academy application, this includes both Claimants and council tax details
```

Comment:
Claimants is a little unclear. We should probably either specify that the data is being retrieved from the Academy Housing Benefits application or make clear that they are housing benefit claimants. 

Claimants doesn't need to be capitalised, Council Tax should be capitalised.

Reply:
Yeah, I think it would be clearer if we mentioned what the type of data we're retrieving from Academy is. e.g council tax, housing benefits